playerUi/timelineMarkersLive.html

```diff
@@ -0,0 +1,178 @@
+      background: -webkit-gradient(left top, right top, color-stop(0%, rgba(44, 131, 185, 1)), color-stop(100%, rgba(30, 171, 227, 1)));
+      background: -webkit-linear-gradient(left, rgba(44, 131, 185, 1) 0%, rgba(30, 171, 227, 1) 100%);
+      background: -o-linear-gradient(left, rgba(44, 131, 185, 1) 0%, rgba(30, 171, 227, 1) 100%);
+      background: -ms-linear-gradient(left, rgba(44, 131, 185, 1) 0%, rgba(30, 171, 227, 1) 100%);
+      background: linear-gradient(to right, rgba(44, 131, 185, 1) 0%, rgba(30, 171, 227, 1) 100%);
+      filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#2c83b9', endColorstr='#1eabe3', GradientType=1);
+    }
+
+    #wrapper {
+      background: url(../images/logo-bg-demopage.png);
+      height: 100vh;
+    }
+
+    #banner {
+      border-bottom: 1px solid #fff;
+      background-color: #1eabe3;
+      width: 100%
+    }
+
+    #banner h1 {
+      margin: 0;
+      padding: 30px;
+    }
+
+    .logo {
+      padding: 10px;
+      width: 25%;
+      min-width: 350px;
+      float: left;
+      margin: auto;
+    }
+
+    .title {
+      width: 75%;
+      white-space: nowrap;
+    }
+
+    .clear {
+      clear: both;
+    }
+
+    .content {
+      margin-bottom: 10em;
+    }
+
+    h1,
+    h2,
+    h3,
+    p {
+      font-weight: 300;
+      text-align: center;
+      margin: 40px;
+    }
+
+    #player {
+      max-width: 900px;
+      width: 90%;
+      margin: auto;
+      -webkit-box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+      -moz-box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+      box-shadow: 0px 0px 56px 0px rgba(0, 0, 0, 0.75);
+    }
+
+    a {
+      color: #97d9ef;
+      font-weight: 400;
+      text-decoration: none;
+    }
+
+    a:hover {
+      color: #fff;
+    }
+
+    @media (max-width: 800px) {
+      .logo {
+        width: 100%;
+      }
+
+      .title {
+        display: none;
+      }
+    }
+  </style>
+</head>
+
+<body>
+  <div id="wrapper">
+    <div id="banner">
+      <div class="logo"><img src="../images/bitmovin-logo.png"></div>
+      <div class="title">
+        <h1>Timeline markers</h1>
+      </div>
+      <div class="clear"></div>
+    </div>
+    <div class="container">
+      <h1>HTML5 Adaptive Streaming Player for MPEG-DASH & HLS</h1>
+      <h2>Your videos play everywhere with low startup delay, no buffering and in highest quality.</h2>
+      <div class="content">
+        <div class="player-wrapper">
+          <div id="player"></div>
+        </div>
+        <div class="description">
+          <p>For more information about the bitmovin player, please have a look at our online <a
+              href="//bitmovin.com/docs" target="_blank">Documentation</a>.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <script type="text/javascript">
+    var conf = {
+      key: "YOUR-PLAYER-LICENSE-KEY",
+      ui: false,
+      playback: {
+        autoplay: true,
+        muted: true,
+      },
```

Comment:
Do we really need this configurations?

Reply:
We could go without it, but markers for live will not be rendered until we start playing. Hence the reason I've set these options. (Muted since asset audio is really annoying :) )

Reply:
What about the `ui`? I think we set it to `false` when we don't want to use ui right? But I think in this case we do want to use the UI, or I am missing something?

Reply:
Well, not really sure how it actually works - have not read the documentation on this. But, if we leave `ui` and we also attach `ui` through `uiManager` we will actually have 2 UI's rendered on the player. You can see same on `timelineMarkersWithUiApi.html`.

Reply:
Oh! Okay. I was also bit confused why it is even working if `UI` is set to false. Make sense 👍